packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -49,6 +63,15 @@ class FlutterGoldenFileComparator implements GoldenFileComparator {
   @visibleForTesting
   final FileSystem fs;
 
+  /// A wrapper for the [dart:io.Platform] API.
+  ///
+  /// This is only for use in tests, where the system platform (the default) can
+  /// be replaced by mock platform instance.
+  @visibleForTesting
+  Platform get platform => _platform;
```

Comment:
this can just be a `final Platform platform` field with `@visibleForTesting`, I think.